htdocs/admin/mails.php

```diff
@@ -554,7 +554,7 @@ function initfields()
 	print '<tr class="liste_titre"><td class="titlefieldmiddle">'.$langs->trans("Parameter").'</td><td>'.$langs->trans("Value").'</td></tr>';
 
 	// Disable
-	print '<tr class="oddeven"><td>'.$langs->trans("MAIN_DISABLE_ALL_MAILS").'</td><td>'.yn($conf->global->MAIN_DISABLE_ALL_MAILS);
+	print '<tr class="oddeven"><td>'.$langs->trans("MAIN_DISABLE_ALL_MAILS").'</td><td>'.(isset($conf->global->MAIN_DISABLE_ALL_MAILS) ? yn($conf->global->MAIN_DISABLE_ALL_MAILS) : '');
```

Comment:
Result is not similar

Reply:
What is print after <td> in older version, when $conf->global->MAIN_DISABLE_ALL_MAILS is not set ?

Reply:
With old code, if  $conf->global->MAIN_DISABLE_ALL_MAILS  is not set, result is
print ...<td>.yn(unset value);

And the function yn is executed. With your change, the function is not executed.